en/includes/guides/authentication/sso-integrations/add-zoom-template.md

```diff
@@ -0,0 +1,69 @@
+# Log in to Zoom using {{ product_name }}
+
+This page guides you through setting up {{ product_name }} to add SSO (Single Sign-On) login to a Zoom organization.
+
+!!! note
+    Ensure you have a Zoom account and your account supports SSO by referring to the [Zoom documentation](https://support.zoom.com/hc/en/article?id=zm_kb&sysparm_article=KB0060673){:target="_blank"}.
+
+## Create the Zoom Service Provider
```

Comment:
```suggestion
## Create the Zoom Application
```

Reply:
Fixed with https://github.com/wso2/docs-is/pull/4893/commits/32b3f3ec55479adc8d3da2b7042bd292a7b20ce1